drivers/adc/ad400x/ad400x.h

```diff
@@ -79,6 +79,8 @@ struct ad400x_dev {
 	uint32_t reg_access_speed;
 	/* Device Settings */
 	enum ad400x_supported_dev_ids dev_id;
+	/* Voltage reference */
+	uint32_t ref_voltage_mv;
```

Comment:
if you add this new parameter, i would add also it's usage in the same commit. Moreover, where do you pass the init param for this to the device descriptor? I see dev->ref_voltage_mv is used in the iio driver.

Reply:
the ref_voltage_mv shoud have been copied to the dev descriptor on the driver, but this got deleted in one of my manipulations. will fix.  and ill squash it o the iio driver patch so that the usage is included.